build.cmd

```diff
@@ -404,13 +404,8 @@ if NOT DEFINED PYTHON (
     exit /b 1
 )
 
-if /i "%__DoCrossArchBuild%"=="1" (
-    set __IntermediatesIncDir=%__CrossCompIntermediatesDir%\src\inc
-    set __IntermediatesEventingDir=%__CrossCompIntermediatesDir%\eventing
-) else (
-    set __IntermediatesIncDir=%__IntermediatesDir%\src\inc
-    set __IntermediatesEventingDir=%__IntermediatesDir%\eventing
-)
+set __IntermediatesIncDir=%__IntermediatesDir%\src\inc
+set __IntermediatesEventingDir=%__IntermediatesDir%\eventing
```

Comment:
What's the purpose of this change? It puts all intermediates in the same place, which seems wrong.

Reply:
ok, I see, you need to do both but were previously only doing one. Didn't see the context before.

---
In reply to: [158347058](https://github.com/dotnet/coreclr/pull/15604#discussion_r158347058) [](ancestors = 158347058)

Reply:
Essentially the idea is that I will write the generated source to the normal intermediate directory on every build, and additionally to the crossgen folder on cross arch builds. This is related to the question I ask earlier of "Do they need to be in both places within the same build?" Before I was doing either or